Apps/AdminWebClient/src/ClientApp/src/services/restUserFeedbackService.ts

```diff
@@ -58,4 +62,96 @@ export class RestUserFeedbackService implements IUserFeedbackService {
                 });
         });
     }
+
+    public getAllTags(): Promise<AdminTag[]> {
+        return new Promise((resolve, reject) => {
+            const headers: Dictionary<string> = {};
+            headers[this.contentTypeHeader] = this.contentType;
+            this.http
+                .get<AdminTag[]>(`${this.BASE_URI}/Tag`, headers)
+                .then((requestResult) => {
+                    if (requestResult) {
```

Comment:
most likely this condition should be almost always true, we do have a status of the request that must be checked

Reply:
fair point, I will remove it in the coming review that ties the front end and backend